modules/globebrowsing/src/globelabelscomponent.cpp

```diff
@@ -50,7 +50,7 @@ namespace {
 
     constexpr const char* KeyLabelsFileName = "FileName";
 
-    constexpr const double LabelFadeRangeConst = 1500.0;
+    constexpr const double LabelFadeOutLimitAltitude_meters = 25000.0;
```

Comment:
Would prefer: `LabelFadeOutLimitAltitudeMeters` to keep the same camelCase naming convention

Reply:
Changed this.